documentation/src/docs/basic-reader-and-writer.md

```diff
@@ -331,3 +331,21 @@ while (events.hasNext())
 }
 
 ```
+getNextStreamCut API:  For a streaming application like Spark, which uses micro-batch reader connectors to read Pravega Streams in parallel batches, needs a streamCut till what the batch can read.
+This api provides a streamCut that is in a location that has not yet been read but is at a bounded distance from the reader's current location. This api takes a starting streamCut and an approximate distance in bytes and return a new stream cut. No segments from the starting streamCut is skipped over. So the cut is either present inside the segments of the starting streamCut or at the boundry of the segments and it may advance to the successor segments if it is at the tail.
```

Comment:
It may be useful to show the API first.
The stream cut is a bounded distance from the startingStreamCut.